api/src/main/java/com/valtech/aem/saas/api/resource/PathTransformer.java

```diff
@@ -10,24 +10,42 @@
 
   /**
    * Gets one or more externalized links for the passed path argument. It retrieves more than one item in case an
-   * internal path can be represented by more than 1 url. (e.g. for indexing)
+   * internal path can be represented by more than one url. (e.g. for indexing). Links are appended with the html
+   * extension
```

Comment:
Can you change wording?
`Gets one or more externalized links for the passed path argument. It returns more than one URL in case a path can be represented by more than one url (e.g. for indexing). In the core implementation a path to an AEM page will be appended with the html extension. For custom rewriting requirements, e.g. extensionless URLs, a custom PathTransformer which higer service ranking is required. See {@link <reference to PathTaransformer>} for details`